src/Applications/LDAS_App/GEOSldas_LDAS.rc

```diff
@@ -123,6 +129,9 @@ FIRST_ENS_ID: 0
 #      Low values for NUM_SGMT are recommended for run-time and storage efficiency.
 #      Default is 1.
 #
+##  for adas-coupling exp ( COUPLING =1 )
+##  always use the fixed  NUM_SGMT: 1 
```

Comment:
Add check into setup routine that asserts that NUM_SGMT is 1 when COUPLING=1.
Exit setup with message that clarifies this.

Reply:
moved the text to LADAS COUPLING EXP INFO in ldas_setup.
@weiyuan-jiang, could you add the check/exit for this in ldas_setup ?

Reply:
it will reset the value